ldcbench.http-node/src/main/java/org/dice_research/ldcbench/nodes/http/simple/dump/comp/TarArchiver.java

```diff
@@ -0,0 +1,53 @@
+package org.dice_research.ldcbench.nodes.http.simple.dump.comp;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import org.apache.commons.compress.archivers.tar.TarArchiveOutputStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TarArchiver extends Archiver {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TarArchiver.class);
+
+    private static final String TAR_MEDIA_TYPE = "application/x-tar";
+    private static final String TAR_FILE_NAME_EXTENSION = ".tar";
+    CompressionStreamFactory compressionFactory = null;
+
+    /**
+     * Create a simple Tar Archiver
+     */
+    public TarArchiver() {
+        mediaType = TAR_MEDIA_TYPE;
+        fileNameExtension = TAR_FILE_NAME_EXTENSION;
```

Comment:
It seems like the media type and the file name extensions should always be defined within the constructor. However, your current implementation does not make it clear. For example, if I would start to implement another archiver, I wouldn't be made explicitly aware of this necessity. You can do that very easily by defining a constructor in your abstract class that takes these two values. I would even suggest that it should be the only constructor to show that it is necessary to set the values.